ui/components/app/transaction-activity-empty-state/transaction-activity-empty-state.tsx

```diff
@@ -1,35 +1,136 @@
-import React from 'react';
+import React, { useCallback } from 'react';
```

Comment:
Duplicating functionality from swaps button in homepage https://github.com/MetaMask/metamask-extension/blob/67755fb42639cb451472c252b0c7705c4555a8ee/ui/components/app/wallet-overview/coin-buttons.tsx#L344-L396